binding/android/cobra/build.gradle

```diff
@@ -40,3 +40,10 @@ if (file("${rootDir}/publish-mavencentral.gradle").exists()) {
 dependencies {
     implementation fileTree(dir: "libs", include: ["*.jar"])
 }
+
+task copyLibs(type: Copy) {
+    from("${rootDir}/../../lib/android")
+    into("${rootDir}/cobra/src/main/jniLibs")
+}
```

Comment:
I like that we do this now, but can we remove the libs from the project, at them to the ignore and add gitkeeps to the dirs? If we do this, there's just not really a reason to have the duplicate libs in the android project